src/test/scala/org/hammerlab/guacamole/LociSetSuite.scala

```diff
@@ -162,6 +160,13 @@ class LociSetSuite extends GuacFunSuite with Matchers {
     set1.union(set2).toString should equal("chr1:40-43")
   }
 
+  sparkTest("loci set parsing with contig lengths") {
+    LociSet.parse(
+      "chr1,chr2,17,chr2:3-5,chr20:10-20",
+      Some(Map("chr1" -> 10, "chr2" -> 20, "17" -> 12, "chr20" -> 5000))).toString should equal(
+        "17:0-12,chr1:0-10,chr2:0-20,chr20:10-20")
```

Comment:
I assume I'm misunderstanding what's happening here, but should the output not be `chr2:3-5` instead of `chr2:0-20`?